### PR TITLE
Raised payload size limit in nodejsaction proxy

### PR DESCRIPTION
**Pull request by psuter**
Added a test that ensures "large" source files can be processed as actions.

### CODE REVIEW

**Comment by psuter** [comment]
PG1/543 on its way

File: core/nodejsActionBase/app.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ var logger  = require('./src/logger').getLogger('logs/nodejsaction.log', 'nodejs
 var service = require('./src/service').getService(config, logger);
 
 app.set('port', config.port);
-app.use(bodyParser.json());
+app.use(bodyParser.json({ limit: "48mb" }));
```
Comment (psuter) [inline core/nodejsActionBase/app.js]: FYI: https://github.com/expressjs/body-parser#limit

Comment (csantanapr) [inline core/nodejsActionBase/app.js]: ha good find @psuter 👍  `Defaults to '100kb'`

**Comment by csantanapr** [comment]
@psuter other runtimes OK?, only nodejs had this problem and missing test

**Pull request merged**